dubbo-metadata/dubbo-metadata-api/src/main/java/org/apache/dubbo/metadata/report/MetadataReportInstance.java

```diff
@@ -53,7 +53,7 @@ public static void init(MetadataReportConfig config) {
                     .removeParameter(METADATA_REPORT_KEY)
                     .build();
         }
-        url = url.addParameterIfAbsent(APPLICATION_KEY, ApplicationModel.getApplicationConfig().getName());
```

Comment:
`ApplicationModel.getApplicationConfig().getName()` is actually the called `ApplicationModel.getName()`